blake2/src/lib.rs

```diff
@@ -1,6 +1,7 @@
 extern crate blake2;
 extern crate ewasm_api;
 
+#[cfg(not(test))]
 use blake2::{Blake2b, Digest};
```

Comment:
The code doesn't seem to use `Digest`, one may as well just directly call `blake2::Blake2b::default()` below and remove these two lines entirely.

The same applies to most of the files.